Lucy's first Website

LucyMeyy opened the pull request:
This website is about a "Verein" we created called "Chefinnen". 
-I'm still trying to shift the "C" logo under "Über Uns" to the top left and fix it to the menu/navigation bar.
- I'll probably also want to add a different header image for each html site/rider. 
- Unsure why there is a bit of a margin on the left of the header / menu.

LucyMeyy committed:
Adjusted fixed Menu, first "Über Uns" pictures.

Still trying to shift the "C" logo under "Über Uns" to the top left and fix it to the menu/navigation bar.

MichelLaufer commented on Code Chefinne/Events.html:
```diff
@@ -0,0 +1,30 @@
+    <link rel="stylesheet" type="text/css" href="style.css"/>
+</head>
+<header>
+    <menu>
+        <a href="./Home.html">Home</a>
+        <a href="./Über Uns.html">Über Uns</a>
+        <a href="./Events.html">Events</a>
+        <a href="./News.html">News</a>
+        <a href="./Inspiration & Empfehlungen.html">Inspiration & Empfehlungen</a>
+      </menu>
```
Really nice menu! 👍 
Nice that you have included it in each of the HTML-files.

I got an error when I tried to click on the link for Über Uns on my computer. 
Maybe try to change the file-name for example to UberUns.html both in the code in the HTML-files
and in the folder.

MichelLaufer commented on Code Chefinne/Home.html:
```diff
@@ -0,0 +1,38 @@
+    <!--Kommentar für meine Übersicht oder Struktur-->
+    
+    <div>
+        Auf <a href="https://www.linkedin.com/groups/8967842/">LinkedIn</a> und WhatsApp teilen wir alles, was uns bewegt.
```
A suggestion when using links is to use noopener noreferrer, for example like:
`Auf <a href="https://www.linkedin.com/groups/8967842/" target="_blank" rel="noopener noreferrer">LinkedIn</a>`

MichelLaufer commented on Code Chefinne/Home.html:
```diff
@@ -0,0 +1,38 @@
+        Auf <a href="https://www.linkedin.com/groups/8967842/">LinkedIn</a> und WhatsApp teilen wir alles, was uns bewegt.
+    </div>
+    <div id="centerAll">
+        <iframe width="560" height="315" src="https://www.youtube.com/embed/E9dKZSG4qNQ" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Nice with the video! 👍 
A suggestion is to split the code into multiple lines, for example:
`<iframe`
`width="560"`
`height="315"`
...
`allow="..."`
`allowfullscreen`
`</iframe>`

MichelLaufer commented on Code Chefinne/Inspiration & Empfehlungen.html:
```diff
@@ -0,0 +1,35 @@
+      </menu>
+    <div>
+        <h1>
+        <img src="Images/Logo C.jpeg" alt="Chefinnen Logo" style="width:10%;height:10%;border-radius:50%;"/>
```
Nice that you use an alt-tag! Important for accessibility. 
I would suggest moving the styling to the CSS-file. For example you can have `<img class="image-logo">` and then move the styling to the CSS-file under `.image-logo`

MichelLaufer commented on Code Chefinne/Inspiration & Empfehlungen.html:
```diff
@@ -0,0 +1,35 @@
+    <div class="grey-box">
+        <div>
+            <h1>Bilder</h1>
+            <img src="https://is1-ssl.mzstatic.com/image/thumb/Podcasts113/v4/62/a5/77/62a5775b-3a6e-2ed0-5766-9ad9925a88bf/mza_1233407614861866600.jpg/1200x1200bb.jpg">
```
A suggestion is to include an alt-tag for the image.

MichelLaufer commented on Code Chefinne/Inspiration & Empfehlungen.html:
```diff
@@ -0,0 +1,35 @@
+        <div>
+            <h1>Videos</h1>
+            <div id="centerAll">
+                <iframe width="560" height="315" src="https://www.youtube.com/embed/E9dKZSG4qNQ" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
A suggestion is to split the code into multiple lines.

MichelLaufer commented on Code Chefinne/Über Uns.html:
```diff
@@ -0,0 +1,43 @@
+    <link rel="stylesheet" type="text/css" href="style.css"/>
+</head>
+<div>
+    <img src="Images/Logo C.jpeg" alt="ChefinnenLogo" style=width:10%;height:100%;border-radius:50%;z-index:200/>
```
A suggestion is to split the code to multiple lines (for src and alt) and to move the styling (e.g. width and height) to the CSS-file, for example using the class "image-logo"

MichelLaufer commented on Code Chefinne/style.css:
```diff
@@ -0,0 +1,88 @@
+@import url('https://fonts.googleapis.com/css2?family=Epilogue:wght@100&display=swap');
+
+
+header{
+    background-image: url(images/Logo\ Chefinne.jpeg);
+    background-size: cover;
+    height: 300px;
+    padding: 0px;
+    margin: 0px;
+}
```
Looks nice! 👍

MichelLaufer commented on Code Chefinne/style.css:
```diff
@@ -0,0 +1,88 @@
+
+body {font-family: 'Epilogue', sans-serif;}
+
+menu {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    font-family: Arial, Helvetica, sans-serif;
+    position: fixed;
+    height: 10%;
+    top: 0%;
+    margin: 0%;
+    width:100%;
+    background-color: rgb(222, 228, 222);
+    background-size: 
+  }
```
The menu looks very good! 
Maybe delete `background-size`?

MichelLaufer reviewed (commented):
Wow, really nice job Lucy! 👍 
I think that both the code and the website looks very nice.
Both the HTML and CSS code are well-structured and easy to follow. 

A general comment is to move some of the styling from HTML to CSS
for example for some of the images. I have added more specific comments below.

Keep up the good work!

LucyMeyy committed:
Added Media Quries to Home, Uber Uns and News. 

Continued to work on Uber Uns (pictures round) and news.

LucyMeyy committed:
28.10: added media queries to Home, Uber Uns, News

LucyMeyy commented:
Hey Michel, thank you so much for your detailed reply, I really appreciated it. Sorry for my late reply, I only realized now that I can reply directly on github ;). 

I've updated the same project (mainly added media queries), but I unfortunately added it twice and am having trouble finding the delete button for the first part.  "28.10.: added media queries to Home, Uber Uns, News" is the correct one (they are both the same, but the first ones I uploaded all individually instead of just dragging the folder into github). Hope this is clear and thank you for your feedback!

LucyMeyy committed:
Added Media Queries & a contact form under "Home"
